ouroboros-consensus/src/unstable-consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Adversarial.hs

```diff
@@ -576,15 +786,42 @@ withinYS (Delta d) !mbYS !(RI.Race (C.SomeWindow Proxy win)) = case mbYS of
 --
 -- The count will be strictly smaller than the number of active slots in the given 'ChainSchema'.
```

Comment:
I think you can remove this line now

> T`he count will be strictly smaller than the number of active slots in the given 'ChainSchema'.`

since we're only ensuring `k+1` after the intersection, instead of only ensuring `1`.

Reply:
Removed in 6da11a8